python/pyspark/sql/tests/test_readwriter.py

```diff
@@ -141,6 +141,25 @@ def count_bucketed_cols(names, table="pyspark_bucket"):
                 .mode("overwrite").saveAsTable("pyspark_bucket"))
             self.assertSetEqual(set(data), set(self.spark.table("pyspark_bucket").collect()))
 
+    def test_insert_into(self):
+        df = self.spark.createDataFrame([("a", 1), ("b", 2)], ["C1", "C2"])
+        df.write.saveAsTable("test_table")
```

Comment:
I think this API requires Hive enabled. Can you check if this test works when we provide no Hive profile? If it doesn't work, we should add a test like `HiveContextSQLTests` in `test_context.py`.